crates/bdk/src/descriptor/template.rs

```diff
@@ -107,7 +109,9 @@ impl<K: IntoDescriptorKey<Legacy>> DescriptorTemplate for P2Pkh<K> {
 ///
 /// let key =
 ///     bitcoin::PrivateKey::from_wif("cTc4vURSzdx6QE6KVynWGomDbLaA75dNALMNyfjh3p8DRRar84Um")?;
-/// let mut wallet = Wallet::new_no_persist(P2Wpkh_P2Sh(key), None, Network::Testnet)?;
+/// let key2 =
```

Comment:
Isn't it better to give better names than `key` and `key2`?

Reply:
https://github.com/bitcoindevkit/bdk/pull/1390#discussion_r1545445207